examples/vpc/main.tf

```diff
@@ -5,7 +5,7 @@
  */
 
 terraform {
-  required_version = "~> 1.4.6"
+  required_version = "1.4.6"
```

Comment:
It's not a good idea to set fixed terraform version. Btw fixing example it's not in the scope of the PR